docs/src/main/asciidoc/datasource.adoc

```diff
@@ -430,26 +431,23 @@ AgroalDataSource inventoryDataSource;
 ----
 
 [[datasource-active]]
-=== Activate/deactivate datasources
+=== Activate or deactivate datasources
 
-If a datasource is configured at build time,
-by default it is active at runtime,
-that is Quarkus will start the corresponding JDBC connection pool or reactive client on application startup.
+When a datasource is configured at build time, it is active by default at runtime.
+This means that Quarkus will start the corresponding JDBC connection pool or reactive client when the application starts.
 
 To deactivate a datasource at runtime, set `quarkus.datasource[.optional name].active` to `false`.
-Then Quarkus will not start the corresponding JDBC connection pool or reactive client on application startup.
-Any attempt to use the corresponding datasource at runtime will fail with a clear error message.
+Quarkus will then skip starting the JDBC connection pool or reactive client during application startup.
+Any attempt to use the deactivated datasource at runtime will result in a clear error message.
```

Comment:
Saying that it results in an error message may be misleading, because then one might think that it will only print a message into the log. But it actually throws an exception - see https://github.com/quarkusio/quarkus/blob/3.15.0/extensions/datasource/common/src/main/java/io/quarkus/datasource/common/runtime/DataSourceUtil.java#L53

So my suggestion would be

> Any attempt to use the deactivated datasource at runtime results in an exception.

Reply:
great @jmartisk , applying!